package.json

```diff
@@ -15,11 +15,11 @@
 	"scripts": {
 		"build": "npm run build --workspace=wrangler --workspace=jest-environment-wrangler --workspace=pages-plugin-example --workspace=wranglerjs-compat-webpack-plugin",
 		"check": "run-p check:* --aggregate-output --continue-on-error",
-		"check:format": "prettier packages/** .changeset --check --ignore-unknown",
+		"check:format": "prettier packages/** .changeset fixtures/** --check --ignore-unknown",
 		"check:lint": "eslint \"packages/**/*.[tj]s?(x)\" --cache --cache-strategy content --max-warnings=0",
 		"check:type": "npm run check:type --workspaces --if-present",
 		"fix": "npm run prettify && npm run check:lint -- --fix",
-		"prettify": "prettier packages/** .changeset --write --ignore-unknown",
+		"prettify": "prettier packages/** .changeset fixtures/** --write --ignore-unknown",
```

Comment:
```suggestion
		"prettify": "prettier . --ignore-path .gitignore --write --ignore-unknown",
```

Reply:
Looked at applying this, but it's running prettier on thousands of files under packages/wrangler-devtools, because it bundles a bunch of node modules with it. Should probably look at ignoring these? Otherwise it's going to run on more than 3000 files every time a change is made.

Reply:
Should we use `--ignore-path .prettierignore` instead?

Reply:
Ah, didn't realize we already had a `.prettierignore`. We can just omit it. `--ignore-path .prettierignore` is the default.

Reply:
```suggestion
		"prettify": "prettier . --write --ignore-unknown",
```

Reply:
This now just adds these four extra files (and the `fixtures` directory), which seems good to me:

```txt
[warn] .github/ISSUE_TEMPLATE/bug-template.yaml
[warn] .github/ISSUE_TEMPLATE/config.yml
[warn] .prettierrc
[warn] .vscode/extensions.json
```